data/5e/wizards_of_the_coast/core/players_handbook/ph_classes.lst

```diff
@@ -313,7 +313,7 @@ CLASS:Paladin	SPELLSTAT:CHA	SPELLTYPE:Divine	BONUS:CASTERLEVEL|Paladin|CL
 20	CAST:0,4,3,3,2
 
 
-CLASS:Ranger	HD:10		TYPE:PC.Base	DEFINE:RangerLVL|0	BONUS:VAR|RangerLVL|CL	BONUS:ABILITYPOOL|Ranger Skill|1	DEFINE:RangerSkillCount|0	BONUS:VAR|RangerSkillCount|1	BONUS:VAR|SpellCasterTotalLVL|RangerLVL/2	BONUS:VAR|SpellCasterClassCount|1
+CLASS:Ranger	HD:10		TYPE:PC.Base	DEFINE:RangerLVL|0	BONUS:VAR|RangerLVL|CL	BONUS:ABILITYPOOL|Ranger Skill|3	DEFINE:RangerSkillCount|0	BONUS:VAR|RangerSkillCount|1	BONUS:VAR|SpellCasterTotalLVL|RangerLVL/2	BONUS:VAR|SpellCasterClassCount|1
 CLASS:Ranger	PREMULT:1,[!PRETEMPLATE:1,First Level],[PREABILITY:1,CATEGORY=Internal,Multiclass Allowed ~ Ranger],[PRECLASS:1,Ranger=1]
```

Comment:
Haven't worked with a multiclass character as of yet, but a 1st level Ranger gets 3 skills choices (pg 90).